entity/events.go

```diff
@@ -0,0 +1,35 @@
+package entity
+
+import "github.com/maas/gomaasclient/entity/event"
+
+// Event represent an event for MAAS node
+type Event struct {
+	UserName    string `json:"username,omitempty"`
+	Node        string `json:"node,omitempty"`
+	Hostname    string `json:"hostname,omitempty"`
+	Created     string `json:"created,omitempty"`
+	Type        string `json:"type,omitempty"`
+	Description string `json:"description,omitempty"`
+	Level       string `json:"level,omitempty"`
```

Comment:
As well:

```suggestion
	Level       event.LogLevel `json:"level,omitempty"`
```

Reply:
plus the `go fmt` required after this suggestion

Reply:
if we don't keep some type for Level, any string will be accepted and the user needs to pass the value explicitly